core/storage/question/gae_models.py

```diff
@@ -261,8 +261,9 @@ def get_question_skill_links_equidistributed_by_skill(
             list(QuestionSkillLinkModel). A list of QuestionSkillLinkModels
                 corresponding to given skill_ids, with
                 total_question_count/len(skill_ids) number of questions for
-                each skill. If not enough questions for a skill, just return
-                all questions it links to.
+                each skill. If not evenly divisable, it will be rounder up.
```

Comment:
divisable --> divisible
rounder --> rounded

What about ordering? Explain it here too (both in terms of order of skills and order of questions within a skill).

Reply:
Done.